semgrep-core/src/parsing/Parse_rule.ml

```diff
@@ -814,8 +815,10 @@ let parse file =
             let n = Sys.command cmd in
             if n <> 0 then failwith (spf "error executing %s" cmd);
             Json_to_generic.program (Parse_json.parse_program tmpfile))
+    | FT.Config FT.Yaml -> Yaml_to_generic.program file
     | _ ->
-        failwith
-          (spf "wrong rule format, only JSON/YAML/JSONNET are valid:%s:" file)
+        logger#error "wrong rule format, only JSON/YAML/JSONNET are valid";
```

Comment:
If we expect this to happen when using a named pipe, then I don't think this should be logged as an error, but rather as info.

Reply:
true, but also I don't expect people to abuse semgrep-core to do what semgrep is supposed to do, so using named pipes for -config is really not the usual case.